crates/swift-integration-tests/src/enum_attributes/already_declared.rs

```diff
@@ -16,10 +16,18 @@ mod ffi2 {
     enum AlreadyDeclaredEnumTest {}
 
     extern "Rust" {
-        fn reflect_already_declared_enum(arg: AlreadyDeclaredEnumTest) -> AlreadyDeclaredEnumTest;
+        fn rust_reflect_already_declared_enum(
+            arg: AlreadyDeclaredEnumTest,
+        ) -> AlreadyDeclaredEnumTest;
+    }
+
+    extern "Swift" {
+        fn swift_reflect_already_declared_enum(
+            arg: AlreadyDeclaredEnumTest,
+        ) -> AlreadyDeclaredEnumTest;
```

Comment:
@chinedufn 

I guess that this function isn't called from anywhere.

Reply:
Ah yeah we should call it. Great point. Want to tell @conectado how to best call it?

Reply:
I added a test [here](https://github.com/chinedufn/swift-bridge/pull/226/commits/7415b9108beb87de95cd4687c1f841ae7b37ea6f) that I think does this.

I couldn't run the tests because I think the CI is failing so lmk if this work or I should do something else?

Reply:
Hi, @conectado 
The CI failure is due to #213. If you want to know more detail, please see #225.

I guess it should be fine as long as you run the test multiple times and then the test pass at least once.

Reply:
Locally it's consistently failing with:

> Testing failed:
>         field has incomplete type 'struct __swift_bridge__$tuple$I32ResultTestOpaqueRustTypeString'
>         failed to emit precompiled header '/Users/conectado/Library/Developer/Xcode/DerivedData/SwiftRustIntegrationTestRunner-cocipvlohdpefifldegpkicaiuld/Build/Intermediates.noindex/PrecompiledHeaders/BridgingHeader-swift_4X3E80DLUM90-clang_KV4PBZK8M8GA.pch' for bridging header '/Users/conectado/repos/firezone/swift-bridge/SwiftRustIntegrationTestRunner/Headers/BridgingHeader.h'
>         Testing cancelled because the build failed

Reply:
This error is due to #213.

>Locally it's consistently failing with:

It is not consistently. Right now, when we run `./test-swift-rust-integration.sh`, The tests will sometimes fails. If you check #225, You could understand why the tests sometimes fails.

On the other hand, when we run `cargo test --all`, the tests will consistently pass.